scripts/zones/Waughroon_Shrine/IDs.lua

```diff
@@ -86,6 +86,50 @@ zones[xi.zone.WAUGHROON_SHRINE] =
             17367285,
         },
     },
+
+    princesses =
+    {
+        [1] =
+        {
+            17367174,
```

Comment:
since they are all sequential in the 3 sets, could we possibly use just the 1st of each set and offset from there?

Reply:
actually, could just offset from the queens IDs